profile-widget/profilewidget2.cpp

```diff
@@ -96,6 +96,14 @@ static struct _ItemPos {
 static const double thumbnailBaseZValue = 100.0;
 #endif
 
+template<typename T, class... Args>
+T *ProfileWidget2::createItem(const DiveCartesianAxis &vAxis, int vColumn, Args&&... args)
+{
+	T *res = new T(*dataModel, *timeAxis, DivePlotDataModel::TIME, vAxis, vColumn,
+		       std::forward<Args>(args)...);
```

Comment:
`std::forward<>` does more than its name suggests.

The thing is that there are numerous ways of passing data from one function to the other (by value, by reference, by rvalue reference). The latter means that the caller is fine with the callee taking ownership of the data (i.e. move semantics):
```
        f(std::move(v)); // <-- poof, the vector may be empty (moved-from) after the call, because callee took ownership.
```

The gist of `std::forward(x)` is that it does the right thing depending on the type of x. It's mostly irrelevant in our case, because we don't use any of these subtleties, but I reckon it is "good style".